Shared/Flows/Avatar Profile/AvatarProfileViewController.swift

```diff
@@ -42,7 +42,7 @@ class AvatarProfileViewController: ViewController {
                 self.nameLabel.setText(user?.fullName)
                 self.focusLabel.setText(user?.focusStatus?.rawValue)
                 self.view.layoutNow()
-            }.add(to: self.taskPool)
+            }.add(to: self.autocancelTaskPool)
```

Comment:
Changing the name to make its behavior more clear. It automatically cancels tasks when the view disappears.